src/modules/displaymanager/displaymanager.schema.yaml

```diff
@@ -25,6 +25,7 @@ properties:
         properties:
             greeter_user: { type: string }
             greeter_group: { type: string }
+            greeter_css_location: {type: string}
```

Comment:
Please stick to the formatting of the other schema lines (e.g. more spaces -- Calamares is a very spacey codebase)